mavlink/tests/agnostic_decode_test.rs

```diff
@@ -75,7 +75,7 @@ mod test_agnostic_encode_decode {
     }
 }
 
-#[cfg(all(feature = "std", feature = "tokio-1", feature = "common"))]
+#[cfg(all(feature = "tokio-1", feature = "common"))]
```

Comment:
Tokio without std would fail if we are not enabling std automatically on tokio feature.